Makefile

```diff
@@ -133,15 +133,19 @@ ${C_BINDING_DIRS}:
 	qmake -o $@Makefile $@qmake.pro
 	cd $@; $(MAKE)
 
-# execute all tests, both command-line tools holochain and "C" bindings
-test: test_holochain test_cmd c_binding_tests ${C_BINDING_TESTS}
+# execute all tests: command-line tools, holochain, nodejs container, and "C" bindings
+test: test_holochain test_cmd test_nodejs_container c_binding_tests ${C_BINDING_TESTS}
 
 test_holochain: build_holochain
 	RUSTFLAGS="-D warnings" $(CARGO) test --all --exclude hc
 
 test_cmd: build_cmd
 	cd cmd && RUSTFLAGS="-D warnings" $(CARGO) test
 
+test_nodejs_container: core_toolchain
+	rustup default ${CORE_RUST_VERSION}
+	cd nodejs_container && yarn install --ignore-scripts && node ./publish.js
```

Comment:
So I don't have yarn on my machine, which means we have to add the yarn install into the tools installation make step:

```
cd nodejs_container && yarn install --ignore-scripts && node ./publish.js
/bin/sh: 1: yarn: not found
Makefile:146: recipe for target 'test_nodejs_container' failed
```

or is there some way around this dependency?

Reply:
there's no way around it... not easily anyways, the package was depending quite a bit on yarn for things

including in the node ./publish.js scripts

Reply:
this only affects the dev team, or people developing holochain... at least